core/domain/exp_domain.py

```diff
@@ -541,10 +689,16 @@ def __init__(self, outcome, rule_specs, correct):
         self.correct = correct
 
     def validate(self, interaction, exp_param_specs_dict):
-        """Rule validation.
+        """Rule validation. Verifies that all rule classes are valid, and that
+        the AnswerGroup only has one classifier rule.
 
-        Verifies that all rule classes are valid, and that the AnswerGroup only
-        has one classifier rule.
+        Args:
+            exp_param_specs_dict: dict. A dict of specified parameters used in
+                the exploration.
+
+        Raises:
+            ValidationError: One or more attributes of the AnswerGroup are
+            invalid or AnswerGroup contains more than one classifier rule.
```

Comment:
ValidationError: One or more attributes of the AnswerGroup are invalid.
            ValidationError: The AnswerGroup contains more than one classifier rule.